### ISSUE TITLE
Vagrantfile problems

### ISSUE THREAD

**Issue by a-nldisr** [mesos/storm#61]
Hi, 

There are a few errors in the Vagrantfile
The first error i created a pull request for, the second error:

```
vm:
* Ports to forward must be 1 to 65535
* Ports to forward must be 1 to 65535
* Ports to forward must be 1 to 65535
```

The ports for the slave are now bound to 80000 range in the Vagrantfile, Linux systems do not go past 65535 ports. But maybe its better to run both slave, storm and master on the same vagrant box to prevent overlapping of ports on two machines.

Right after that i got the next error:
```
==> slave: Waiting for machine to boot. This may take a few minutes...
    slave: SSH address: 127.0.0.1:2200
    slave: SSH username: vagrant
    slave: SSH auth method: private key
    slave: Warning: Connection timeout. Retrying...
```

4th error: 
```
==> master: stdin: is not a tty
==> master: grep: 
==> master: /vagrant/pom.xml
==> master: : No such file or directory
==> master: cp: 
==> master: cannot stat ‘/vagrant/storm-mesos-.tgz’
==> master: : No such file or directory
==> master: /tmp/vagrant-shell: line 10: cd: /vagrant/_release/storm-mesos-: No such file or directory
==> master: /tmp/vagrant-shell: line 11: bin/storm-mesos: No such file or directory
```

Also a tip: 
Maybe its better to run mesos-slave + master + storm on the same host to keep the resources on systems reasonable. You are now starting 2x virtualbox with 4CPU, 4G mem per instance. My macbook came to a crawl instantly when these machines started up.

**Comment by erikdw** [comment]
@a-nldisr : thanks for the report.  Breaking it down:

1. I didn't see a PR from you, so not sure what the 1st error was?
2. *ports going over 65535*:  I also noticed this issue and will be correcting it in a PR very soon.
3. *`Warning: Connection timeout. Retrying...`*: this isn't an issue.  It's just standard vagrant behavior, where it polls for the VM's SSH daemon to be ready, and it takes a little bit of time.
4. Seems there is a problem with the virtualbox guest additions feature of files between the host and guest.  `/vagrant` in the guest is supposed to be a mount of the `vboxsf` filesystem, which exposes the files from the "pwd" of the host.  The setup in this Vagrantfile is totally dependent on the file sharing working, and since it isn't seeming to be working maybe that's the problem?  What does `ls /vagrant` within the guest show?  It should have all the files in the basedir of your repo.
5. *launching 2 heavy-weight VMs*:  yeah, it's probably overkill.  The 1st VM (called "master") already has everything (mesos-master, mesos-slave, zookeeper, mesos-nimbus, storm UI).  The 2nd VM is just an additional mesos-slave host.  How about I just disable the 2nd slave by default and have a config option to enable it.

**Comment by erikdw** [comment]
Resolution:

1. Not explicitly stated, so nothing can be done.
2. Fixed in PR:  https://github.com/mesos/storm/pull/80
3. I believe this to be standard vagrant behavior and not an issue
4. Since that seems to be specific to the reporter's environment, I don't think it's a general issue with the vagrant setup of this project.  (I've confirmed that at least 4 separate people are able to run this vagrant setup on their laptops)
5. Fixed in PR:  https://github.com/mesos/storm/pull/80
